protect_archiver/downloader/get_camera_list.py

```diff
@@ -29,6 +29,8 @@ def get_camera_list(session: Any, connected: bool = True) -> List[Camera]:
     for camera in cameras:
         if connected and camera["state"] != "CONNECTED":
             continue
+        if camera["stats"]["video"]["recordingStart"] is None:
+            continue
         camera_list.append(
```

Comment:
Thanks for the suggestion! Let's merge the two statements into one. That way we end up with a single block where future conditions can be easily added as needed.

```suggestion
    for camera in cameras:
        if (
            (connected and camera["state"] != "CONNECTED") or  # ignore disconnected cameras
            (camera["stats"]["video"]["recordingStart"] is None)  # ignore cameras without recordings
        ): 
            continue

        camera_list.append(
```

Reply:
Not sure if it is more readable, but I've merged the conditions now.